dashboard/app/models/lesson.rb

```diff
@@ -376,6 +376,7 @@ def get_uncached_edit_path
       edit_lesson_path(id: id)
   end
 
+  # TODO: TEACH-2049 Should this path use the unit group? Unsure where this is used.
```

Comment:
because we only need this path on levelbuilder as part of the edit lesson workflow, and the lesson will have the same content regardless of which course it's viewed from (or viewed from no course), it seems ok to leave this on /s/ paths, or (if using course urls) to keep using the original unit group. does that help answer the questions? there is also the code comment below which explains where this is used.